earth_enterprise/src/fusion/tools/gevectorquery.cpp

```diff
@@ -53,8 +53,9 @@ int main(int argc, char** argv) {
         argn++;
         if (argn >= argc)
           usage();
-        output = reinterpret_cast<char*>(alloca(strlen(argv[argn]) + 1));
-        strcpy(output, argv[argn]);
+        int outputlen = strlen(argv[argn]);
```

Comment:
For similar reasons this is probably also a size_t, which I think is what auto would give us.  We could now use auto since we are c++11...